src/app/navigation/application-launcher.component.ts

```diff
@@ -0,0 +1,66 @@
+import {
+    Component, ElementRef, EventEmitter,
+    Input, Output, Renderer2, TemplateRef,
+    ViewEncapsulation
+} from '@angular/core';
+
+import { NavigationItemConfig } from './navigation-item-config';
+
+
+
+/**
+ * Application Launcher component
+ * @param {boolean=} isDisabled Disable the application launcher button, default: false
+ * @param {boolean=} isList Display items as a list instead of a grid, default: false
+ * @param {boolean=} hiddenIcons Flag to not show icons on the launcher, default: false
+ */
+
+@Component({
+    encapsulation: ViewEncapsulation.None,
+    selector: 'pfng-application-launcher',
+    styleUrls: ['./application-launcher.component.less'],
```

Comment:
I don't see a less file. Did you forget to commit that?

Reply:
@dlabrecq: yeah, I forget to commit it.

Reply:
Introduced some changes, earlier this week -- please pull the latest.

To allow for customization, we no longer embed the CSS in the component. Instead, we now build the CSS as a single file, under the src/assets/stylesheets directory.

Please remove the styleUrls property.